keytree/src/lib.rs

```diff
@@ -217,6 +249,20 @@ mod tests {
         );
     }
 
+    #[test]
+    fn random_xprv_leaf_test() {
+        let seed = [0u8; 32];
+        let mut rng = ChaChaRng::from_seed(seed);
+        let xprv = Xprv::random(&mut rng).derive_leaf_key(|t| {
+            t.commit_u64(b"account_id", 34);
```

Comment:
let's maybe do a more leaf-like example? Accounts are themselves key-trees, but individual invoices aren't:

```suggestion
            t.commit_u64(b"invoice_id", 1000024);
```

(because paperwork identifiers often long strings of zeroes in the middle ¯\\\_(ツ)_/¯ )

Reply:
oh, yes, good point!